securedrop/tests/test_unit_journalist.py

```diff
@@ -7,15 +7,17 @@
 import mock
 
 from flask_testing import TestCase
-from flask import url_for, escape, g
+from flask import url_for, escape
 
 import crypto_util
 import journalist
 import common
-from db import db_session, Source, Journalist, InvalidPasswordLength
+from db import (db_session, Source, Journalist, InvalidPasswordLength,
+                Submission)
 
 # Set environment variable so config.py uses a test environment
 os.environ['SECUREDROP_ENV'] = 'test'
+import config
```

Comment:
Where is config being called? Or is this simply for making sure the flask settings are set?